conans/test/integration/basic_build_test.py

```diff
@@ -15,16 +15,18 @@ class BasicBuildTest(unittest.TestCase):
 
     def complete_build_flow_test(self):
         """In local user folder"""
-        files = cpp_hello_conan_files("Hello0", "0.1")
-        client = TestClient()
-        client.save(files)
-        command = "say_hello" if platform.system() == "Windows" else "./say_hello"
+        command = "bin\\say_hello" if platform.system() == "Windows" else "./bin/say_hello"
 
-        for install, lang, static in [("install", 0, True),
-                                      ("install -o language=1", 1, True),
-                                      ("install -o language=1 -o static=False", 1, False),
-                                      ("install -o static=False", 0, False)]:
-            client.run(install)
+        for lang, static in [(0, True), (1, True), (1, False), (0, False)]:
```

Comment:
You are changing the purpose of the test. The test is checking wheter a user can issu different install commands with different options, for the same project. If you instantiate a new TestClient(), it is like you are creating a new computer.